src/app/main/connection-window/connection-window.component.ts

```diff
@@ -13,6 +13,7 @@ import { ConnectionWindowService } from './connection-window.service';
   styleUrls: ['./connection-window.component.scss'],
 })
 export class ConnectionWindowComponent extends SubscriptionHandler implements OnInit {
```

Comment:
you need to change this into `implements OnInit, AfterViewInit {` to fix the error below and use ngAgterViewInit properly

Reply:
Thank you, I didn't realize we needed to implements the lifecycle methods.